### PR TITLE
Enable ktlint

### PR DESCRIPTION
**Pull request by danimontoya**
This PR enables ktLint #31 .

### CODE REVIEW

File: arrow-fx/src/test/kotlin/arrow/fx/RefTest.kt [hunk 1]
```diff
@@ -18,7 +18,7 @@ import io.kotlintest.shouldBe
 class RefTest : UnitSpec() {
 
   init {
-    fun <F> MonadDefer<F>.tests(EQF: EqK<F>, RF: RefFactory<F>): Unit {
```
Comment (rachelcarmena) [inline arrow-fx/src/test/kotlin/arrow/fx/RefTest.kt]: Congrats @danimontoya !! You got it!! Now we could undo these changes. They aren't API but an example for other functions :+1:

Comment (danimontoya) [reply arrow-fx/src/test/kotlin/arrow/fx/RefTest.kt]: Sure!

Comment (danimontoya) [reply arrow-fx/src/test/kotlin/arrow/fx/RefTest.kt]: Thanks :)
